docs/types/value-types.rst

```diff
@@ -305,8 +305,7 @@ Contract Types
 
 Every :ref:`contract<contracts>` defines its own type.
 You can implicitly convert contracts to contracts they inherit from.
-Contracts can be explicitly converted to and from all other contract types
-and the ``address`` type.
+Contracts can be explicitly converted to the ``address`` type.
```

Comment:
```suggestion
Contracts can be explicitly converted to and from the ``address`` type.
```

Reply:
thx! done in `3a452181a0d54e47df8640742c906b347392ffd7`